openid-sharedsignals-framework-1_0.md

```diff
@@ -2218,9 +2220,9 @@ endpoint_url
 
 authorization_header
 
-> The HTTP Authorization header that the Transmitter MUST set with each event
-  delivery, if the configuration is present. The value is optional and it is set
-  by the Receiver.
+> If the endpoint_url requires authorization, the receiver is required to provide
```

Comment:
```suggestion
> If the endpoint_url requires authorization, the receiver SHOULD provide
```

Reply:
I am going to call this a MUST to maintain the earlier language